v2/complete-migration.php

```diff
@@ -9,6 +9,13 @@ function onesignal_complete_migration()
             wp_die(__('Security check failed', 'onesignal-push'));
         }
 
+        // Upgrade from v2 to v3 - enable notification_on_page for existing users
+        // This prevents breaking changes for users who were used to v2 behavior
+        $settings = get_option('OneSignalWPSetting', array());
+        $settings['notification_on_page'] = 1;
+
+        update_option('OneSignalWPSetting', $settings, 'no');
+
```

Comment:
Won't the code we added to `onesignal.php` already take care of this?

Reply:
You're right -- it's removed now.